src/main/java/dev/j3fftw/litexpansion/machine/AdvancedSolarPanel.java

```diff
@@ -49,6 +50,8 @@
         "&cNot Generating..."
     );
 
+    String time;
```

Comment:
????

Reply:
I have to initialize it and then its set later on, since i refer to it when setting the gui